pycon/templates/finaid/receipt_form.html

```diff
@@ -0,0 +1,65 @@
+{% extends "site_base.html" %}
+
+{% load bootstrap_tags %}
+{% load compress %}
+{% load static %}
+{% load i18n %}
+{% load boxes_tags %}
+
+{% block head_title %}Receipt Form{% endblock %}
+
+{% block body_id %}receipt_form{% endblock %}
+
+{% block page_title %}Receipt Application{% endblock %}
+
+{% block body %}
+    {% if form.errors %}
+        <div class="alert">
+            <p class="text-error">
+                There were errors in your form, please correct them and submit again.
+            </p>
+        </div>
+    {% endif %}
+
+    <form method="POST" action="" enctype="multipart/form-data" class="form-horizontal">
+        {% csrf_token %}
+        {% if form.errors and not form.non_field_errors %}
```

Comment:
I haven't seen this `if form.errors and not form.non_field_errors` pattern before - what does it do?